webapp/graphite/node.py

```diff
@@ -25,8 +25,13 @@ def __init__(self, path, reader):
     self.reader = reader
     self.is_leaf = True
 
-  def fetch(self, startTime, endTime):
-    return self.reader.fetch(startTime, endTime)
+  def fetch(self, startTime, endTime, now=None, requestContext=None):
+    try:
```

Comment:
It's needed to support readers that don't accept the new parameters, we could go through and update all the readers and tests to accept them, but that would still leave people with 3rd-party readers needing to update them for no good reason.